src/bootstrap/compile.rs

```diff
@@ -500,7 +500,7 @@ impl Step for Rustc {
 
 /// Same as `std_cargo`, but for libtest
 pub fn rustc_cargo(build: &Build,
-                   compiler: &Compiler,
+                   _compiler: &Compiler,
```

Comment:
If this argument remains unused, could it be removed?

Reply:
Yep, I've added commit to remove unused argument.